recipes/hoggorm/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "hoggorm" %}
+{% set version = "0.13.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hoggorm-{{ version }}.tar.gz
+  sha256: f1254d860f3024b2e6a2cecb2c0152f2010596235058781856c3a1a2f075c3b6
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - sphinx
+    - numpy >=1.11.3
```

Comment:
You probably don't need numpy to build the package

Reply:
Thanks for reviewing.
I don't understand that as it's not included in the build-section. Removing it from the host section and the 'staged-recipes (linux_64 linux)' will fail.

Reply:
Are you compiling and linking against numpy?

Reply:
Now, I think understand your point. I've found https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy and also a discussion at staged-recipes. I do not exactly know if it is needed as build-req.
grep -H numpy * shows
README.rst:- numpy >= 1.9
requirements.txt:numpy >= 1.11.3
setup.cfg:	numpy>=1.9

I removed the dependency

Reply:
According to a comment in https://gitter.im/conda-forge/conda-forge.github.io
"Normally it shouldn't be but it looks like the package is importing itself when running setup.py to get the version. It's generally seen as a bad practice and therefore uncommon but in this case you will need numpy in host. It's worth including a comment in the meta.yaml explaining why it's there."